osbrain/nameserver.py

```diff
@@ -69,6 +69,12 @@ def daemon_shutdown(self):
         """
         self._pyroDaemon.shutdown()
 
+    def raise_exception(self):
+        """
+        Raise an exception (for testing purposes).
+        """
+        raise RuntimeError('User raised an exception!')
+
```

Comment:
No need to implement this method now. :wink:

Reply:
Good point